packages/plugin-web-attribution-browser/test/helpers.test.ts

```diff
@@ -125,6 +125,20 @@ describe('isNewCampaign', () => {
 
     expect(isNewCampaign(currentCampaign, previousCampaign, {}, false)).toBe(false);
   });
+
+  test('should return true for no referrer with any new campaign in the same session', () => {
+    const previousCampaign = {
+      ...BASE_CAMPAIGN,
+      utm_campaign: 'utm_campaign',
+      referring_domain: 'a.b.c.d',
```

Comment:
we can also remove this since in practice `referring_domain` exists when a `referrer` exists

Reply:
ohh i misread, this is for the previous campaign. please ignore